pygmt/base_plotting.py

```diff
@@ -1918,3 +1918,206 @@ def update_pointers(data_pointers):
             with file_context as fname:
                 arg_str = " ".join([fname, build_arg_string(kwargs)])
                 lib.call_module("meca", arg_str)
+
+    @fmt_docstring
+    @use_alias(
+        A="vector",
+        B="frame",
+        C="cmap",
+        D="rescale",
+        E="uncertainty_color",
+        G="facecolor",
+        J="projection",
+        L="line",
+        N="no_clip",
+        R="region",
+        S="scaling",
+        U="timestamp",
+        V="verbose",
+        W="pen",
+        X="xshift",
+        Y="yshift",
+        p="perspective",
+        t="transparency",
+    )
+    @kwargs_to_strings(R="sequence", i="sequence_comma")
+    def velo(self, data=None, scaling=None, **kwargs):
+        """
+        Plot velocity vectors, crosses, and wedges
+
+        Reads data values from files, numpy array or pandas DataFrame and will
+        plot velocity arrows on a map. Most options are the same as for plot,
+        except *scaling*.
+
+        Must provide  *data* and *scaling*.
+
+
+        Full option list at :gmt-docs:`supplements/geodesy/velo.html`
+
+        {aliases}
+
+        Parameters
+        ----------
+        data : str or np.ndarray or pandas.DataFrame
+            Pass in either a file name, a 2D numpy array, or a pandas dataframe
+            table.
+
+        {J}
+
+        {R}
+
+        scaling: str
+            Selects the meaning of the columns in the data file and the figure
+            to be plotted. In all cases, the scales are in data units per
+            length unit and sizes are in length units (default length unit is
+            controlled by :gmt-term:`PROJ_LENGTH_UNIT` unless **c**, **i** , or
+            **p** is appended).
+
+            "**e**\\ *velscale/confidence*\\ [\\ **+f**\\ font]"
+
+                Velocity ellipses in (N,E) convention. velscale sets the
+                scaling of the velocity arrows. The confidence sets the
+                2-dimensional confidence limit for the ellipse, e.g., 0.95 for
+                95% confidence ellipse. font sets the font and size of the text
+                [9p,Helvetica,black]. The ellipse will be filled with the color
+                or shade specified by the -G option [default transparent].
+                The arrow and the circumference of the ellipse will be drawn
+                with the pen attributes specified by the -W option. Parameters
+                are expected to be in the following columns:
```

Comment:
Should have mentioned this earlier, I usually copy the docs from GMT (e.g at https://github.com/GenericMappingTools/gmt/blob/6.1/doc/rst/source/supplements/geodesy/velo_common.rst_) rather than the HTML page, and just change `\` to `\\`, make sure short aliases like `-W` are converted to long form `pen`, and wrap the docstrings to 79 characters.

```suggestion
            "**e**\\ *velscale/confidence*\\ [**+f**\\ *font*]"

                Velocity ellipses in (N,E) convention. *velscale* sets the
                scaling of the velocity arrows. The *confidence* sets the
                2-dimensional confidence limit for the ellipse, e.g., 0.95 for
                95% confidence ellipse. *font* sets the font and size of the
                text [9p,Helvetica,black]. The ellipse will be filled with the
                color or shade specified by the *facecolor* option [default
                transparent]. The arrow and the circumference of the ellipse
                will be drawn with the pen attributes specified by the *pen*
                option. Parameters are expected to be in the following columns:
```